src/test/java/com/google/cloud/tools/maven/it/RunAsyncMojoIntegrationTest.java

```diff
@@ -46,22 +44,12 @@ public void initPorts() throws IOException {
     adminPort = SocketUtil.findPort();
   }
 
-  @Test
-  public void testRunAsyncStandardV1()
-      throws IOException, VerificationException, InterruptedException {
-    test("testRunAsyncV1", SupportedDevServerVersion.V1);
-  }
-
-  @Test
-  public void testRunAsyncStandardV2Alpha()
-      throws IOException, VerificationException, InterruptedException {
-    test("testRunAsyncV2Alpha", SupportedDevServerVersion.V2ALPHA);
-  }
-
-  private void test(String name, SupportedDevServerVersion version)
+  private void testRunAsync(String name)
       throws IOException, VerificationException, InterruptedException {
+    String testName = "testRunAsync";
     try {
-      Verifier verifier = createVerifier(name, version);
+      Verifier verifier = createVerifier(testName);
+      verifier.setSystemProperty("app.devserver.port", Integer.toString(serverPort));
```

Comment:
This is done in `createVerifier()` too. Maybe just remove `createVerifier()` and just do `new StandardVerifier(testName)`, or move much of code here to `createVerifier()` if duplicated repeatedly in other tests?

Reply:
Yeah since this is used twice, I've left the extra `createVerifier` in and cleaned it up as you suggested.